### ISSUE TITLE
Amount not parsed

### ISSUE THREAD

**Issue by firepol** [VerenigingCampusKabel/python-sepa#1]
Hi,

I'm parsing Swiss PostFinance camt.053 files (v4), your library works almost fine, except it doesn't parse the amount.

E.g.: in the XML, my balance elements look like this: `<Amt Ccy="EUR">123.00</Amt>`

In your example to get started I modified to get. instead of the mandate_initiation_request, the statement (`bank_to_customer_statement`):

```
data_out = parser.parse_string(parser.bank_to_customer_statement, data_in)
print(data_out)
```

Everything seems to be parsed correctly, except the amount that results in an empty dictionary:

```
'amount': {},
```

I spent already some time and I ended with my own "hack" in the `parser.py` in the `parse_tree` method:

```
    if tag.tag == 'Amt':
        return {'amount': tag.text,
                'currency': tag.attrib['Ccy']}
```

Probably for you a clean fix would be easier to do than for me. If you want I can provide an example of the PostFinance camt file, just let me know.

**Comment by DanielHuisman** [comment]
Could you tell me where in the camt.053 file this occurs? Specifically the nesting of tags, so I can fix the definition file (https://github.com/VerenigingCampusKabel/python-sepa/blob/master/sepa/definitions/statement.py).

**Comment by firepol** [comment]
Hi Daniel, thanks for the answer,

here an example (anonymized a bit) of camt 053 v4 I get from PostFinance:

It happens basically for every "Amt" occurrence.

```
<?xml version="1.0" encoding="UTF-8"?>
<Document xmlns="urn:iso:std:iso:20022:tech:xsd:camt.053.001.04" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="urn:iso:std:iso:20022:tech:xsd:camt.053.001.04 camt.053.001.04.xsd">
    <BkToCstmrStmt>
        <GrpHdr>
            <MsgId>20180131123456789012345</MsgId>
            <CreDtTm>2018-01-31T16:29:00</CreDtTm>
            <MsgPgntn>
                <PgNb>1</PgNb>
                <LastPgInd>true</LastPgInd>
            </MsgPgntn>
            <AddtlInf>Productive</AddtlInf>
        </GrpHdr>
        <Stmt>
            <Id>20180131112233445566778</Id>
            <ElctrncSeqNb>21</ElctrncSeqNb>
            <CreDtTm>2018-01-31T16:29:00</CreDtTm>
            <FrToDt>
                <FrDtTm>2018-01-01T00:00:00</FrDtTm>
                <ToDtTm>2018-01-31T23:59:59</ToDtTm>
            </FrToDt>
            <Acct>
                <Id>
                    <IBAN>CH0409000000123456789</IBAN>
                </Id>
                <Ownr>
                    <Nm>First LastName MyCity</Nm>
                </Ownr>
            </Acct>
            <Bal>
                <Tp>
                    <CdOrPrtry>
                        <Cd>OPBD</Cd>
                    </CdOrPrtry>
                </Tp>
                <Amt Ccy="EUR">123.55</Amt>
                <CdtDbtInd>CRDT</CdtDbtInd>
                <Dt>
                    <Dt>2017-12-31</Dt>
                </Dt>
            </Bal>
            <Bal>
                <Tp>
                    <CdOrPrtry>
                        <Cd>CLBD</Cd>
                    </CdOrPrtry>
                </Tp>
                <Amt Ccy="EUR">0.55</Amt>
                <CdtDbtInd>CRDT</CdtDbtInd>
                <Dt>
                    <Dt>2018-01-31</Dt>
                </Dt>
            </Bal>
            <Ntry>
                <Amt Ccy="EUR">123.00</Amt>
                <CdtDbtInd>DBIT</CdtDbtInd>
                <RvslInd>false</RvslInd>
                <Sts>BOOK</Sts>
                <BookgDt>
                    <Dt>2018-01-08</Dt>
                </BookgDt>
                <ValDt>
                    <Dt>2018-01-08</Dt>
                </ValDt>
                <AcctSvcrRef>20180107000800999888777666555444</AcctSvcrRef>
                <BkTxCd>
                    <Domn>
                        <Cd>PMNT</Cd>
                        <Fmly>
                            <Cd>ICDT</Cd>
                            <SubFmlyCd>ESCT</SubFmlyCd>
                        </Fmly>
                    </Domn>
                </BkTxCd>
                <NtryDtls>
                    <TxDtls>
                        <Refs>
                            <AcctSvcrRef>20180107000800999888777666555444</AcctSvcrRef>
                        </Refs>
                        <Amt Ccy="EUR">123.00</Amt>
                        <CdtDbtInd>DBIT</CdtDbtInd>
                    </TxDtls>
                </NtryDtls>
                <AddtlNtryInf>GIRO INTERNATIONAL (SEPA) SOME ADDRESS
                    SOMEIBAN SOMECOMPANY
                </AddtlNtryInf>
            </Ntry>
            <Ntry>
                <Amt Ccy="EUR">0.00</Amt>
                <CdtDbtInd>DBIT</CdtDbtInd>
                <RvslInd>false</RvslInd>
                <Sts>BOOK</Sts>
                <BookgDt>
                    <Dt>2018-01-31</Dt>
                </BookgDt>
                <ValDt>
                    <Dt>2018-01-31</Dt>
                </ValDt>
                <AcctSvcrRef>20180131001112223334445556667778</AcctSvcrRef>
                <BkTxCd>
                    <Domn>
                        <Cd>PMNT</Cd>
                        <Fmly>
                            <Cd>ICDT</Cd>
                            <SubFmlyCd>CHRG</SubFmlyCd>
                        </Fmly>
                    </Domn>
                </BkTxCd>
                <NtryDtls>
                    <TxDtls>
                        <Refs>
                            <AcctSvcrRef>20180131001112223334445556667778</AcctSvcrRef>
                        </Refs>
                        <Amt Ccy="EUR">0.00</Amt>
                        <CdtDbtInd>DBIT</CdtDbtInd>
                    </TxDtls>
                </NtryDtls>
                <AddtlNtryInf>some bla bla</AddtlNtryInf>
            </Ntry>
        </Stmt>
    </BkToCstmrStmt>
</Document>

```

Please note, that to make it work I need another dirty hack like this in my tool before calling your parser:

```
DOCUMENT_ATTRIBUTES = ' xmlns="urn:iso:std:iso:20022:tech:xsd:camt.053.001.04" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="urn:iso:std:iso:20022:tech:xsd:camt.053.001.04 camt.053.001.04.xsd"'

xml = file.read()
xml = xml.replace(DOCUMENT_ATTRIBUTES, '')
```

If you can fix that too, that would be nice, because I don't see why the `Document` element of the xml should be expected to have no attributes... it's pretty standard to get XMLs with lots of attributes that can be ignored and it should just work fine.

In statement.py I noticed

Here the documentation for the Swiss implementation of camt 053 (as mentioned before we use v4, but I wonder if v6 has the amount used differently?): https://www.six-group.com/interbank-clearing/dam/downloads/en/standardization/iso/swiss-recommendations/implementation-guidelines-camt.pdf

**Comment by DanielHuisman** [comment]
Should be fixed in the latest release v0.5.0 (https://github.com/VerenigingCampusKabel/python-sepa/commit/e44eb32c3c4ce61d6dafa6fbe12a11afb6384076)
